waku/v2/protocol/filter/waku_filter.go

```diff
@@ -62,13 +62,13 @@ type (
 // relay protocol.
 const FilterID_v20beta1 = libp2pProtocol.ID("/vac/waku/filter/2.0.0-beta1")
 
-func NewWakuFilter(ctx context.Context, host host.Host, isFullNode bool, log *zap.SugaredLogger, opts ...Option) (*WakuFilter, error) {
+func NewWakuFilter(ctx context.Context, host host.Host, isFullNode bool, log *zap.Logger, opts ...Option) (*WakuFilter, error) {
 	wf := new(WakuFilter)
-	wf.log = log.Named("filter")
+	wf.log = log.Named("filter").With(zap.Bool("fullNode", isFullNode))
```

Comment:
The `fullNode` attribute is meant to replace the hints in the various log entries emphasizing full vs light node cases. This attribute should now be present on all entries consistently.